### ISSUE TITLE
Allow resetting overridden properties individually for template instances 

### ISSUE THREAD

**Issue by bjorn** [bjorn/tiled#1725]
While it is currently possible to reset a template instance entirely, it should be possible to reset properties like name, type, size and rotation individually. This could be achieved by a small button in the Properties view for each property.

In addition it would be nice to highlight overridden properties (in bold for example).

Note that Qt Designer is doing both these things (featuring a reset button and highlighting changed properties in bold), so its source may give some idea of how to implement this.

A reset button is also already implemented for color properties, though I'm not sure how suitable its implementation is for this case.

**Comment by bjorn** [comment]
@thabetx Did I understand correctly that you already made some progress on this?

**Comment by thabetx** [comment]
@bjorn I didn't plan to implement this feature, but I've experimented with the property browser when I was working on locking properties.

**Comment by UliAbo** [comment]
Is this task here done and can be closed? As we have the Reset buttons now?

**Comment by bjorn** [comment]
> Is this task here done and can be closed? As we have the Reset buttons now?

No, this task is not done because it's not talking about custom properties, but "properties like name, type, size and rotation". It refers to the "Reset Template Instance(s)" context menu action, which resets a template entirely, where it's not possible to just reset its rotation, for example.
